content/docs/services/redis28.md

```diff
@@ -14,7 +14,7 @@ cloud.gov offers [Redis](https://www.redis.io/) 2.8 as a service. **This service
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`standard` | Redis instance with unlimited storage | Free in Alpha
+`standard` | Redis instance with several GB storage | Free in Alpha
```

Comment:
Ah ha! I'd suggest explaining that on https://cloud.gov/docs/services/redis28/ - that it's currently 10 GB but may change.